InitializeRepo/initialize.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+
+git pull > /dev/null
```

Comment:
```suggestion
git pull
```

I would generally recommend avoiding hiding output. In this case it would appear for a time as though the script isn't running, and if troubleshooting were required due to something unexpected happening here it might be some time before someone considers removing the redirect.

Reply:
Removed